kubernetes/helm/index.yaml

```diff
@@ -1,6 +1,34 @@
 apiVersion: v1
 entries:
   pinot:
+  - apiVersion: v1
+    appVersion: 0.2.7
+    created: "2023-05-04T20:38:46.922122-04:00"
+    dependencies:
+    - condition: pinot.zookeeper.enabled,zookeeper.enabled
+      name: zookeeper
+      repository: https://charts.bitnami.com/bitnami
+      version: 9.x.x
+    description: Apache Pinot is a realtime distributed OLAP datastore, which is used
+      to deliver scalable real time analytics with low latency. It can ingest data
+      from offline data sources (such as Hadoop and flat files) as well as online
+      sources (such as Kafka). Pinot is designed to scale horizontally.
+    digest: f466ab18582850c589c0a6fc264e12ee236ab9cb3b540b653822296e2f65a323
+    home: https://pinot.apache.org/
+    keywords:
+    - olap
+    - analytics
+    - database
+    - pinot
+    maintainers:
+    - email: dev@pinot.apache.org
+      name: pinot-dev
+    name: pinot
+    sources:
+    - https://github.com/apache/pinot/tree/master/kubernetes/helmq
+    urls:
+    - pinot-0.2.7.tgz
```

Comment:
i am no expert in helm but do we allow putting binary in the repo? CC @xiangfu0

Reply:
We are already doing it, so I assume this is the standard step